packages/strapi-utils/lib/content-types.js

```diff
@@ -108,6 +108,63 @@ const isMediaAttribute = attr => {
   return (attr.collection || attr.model) === 'file' && attr.plugin === 'upload';
 };
 
+const getKind = obj => obj.kind || 'collectionType';
+
+const pickSchema = model => {
+  const schema = _.cloneDeep(
+    _.pick(model, ['connection', 'collectionName', 'info', 'options', 'attributes'])
+  );
+
+  schema.kind = getKind(model);
+  return schema;
+};
+
+const formatContentType = (
+  model,
+  { modelName, modelOrigin, defaultConnection },
+  { apiName, pluginName } = {}
+) => {
+  const contentType = {
+    __schema__: pickSchema(model),
+    kind: getKind(model),
+    modelType: 'contentType',
+    modelName,
+    globalId: model.globalId || _.upperFirst(_.camelCase(modelName)),
```

Comment:
you could create a toGlobalId util for this :)

Reply:
Apparently the `modelId` is always already set here, so I removed the line :)